<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaMenuPrincipal</class>
 <widget class="QMainWindow" name="ventanaMenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Men√∫ principal del sistema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFondoMenuEliminarPerfil">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>851</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>menuEliminarPerfilFigma.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="labelRealizarEliminacion">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>650</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelVolverInicio">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIdPerfilEliminar">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>660</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewVerPerfiles">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>1741</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfiguracion">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>531</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
